### DISCUSSION TITLE
Publish Script to show Exchange Calendar Diagnostic Logs Summary in Excel

### DISCUSSION THREAD

**Discussion by Shanefe** [dfinke/ImportExcel#1641] [Show and tell]
Understanding Calendar Logs in Exchange Online has always been hard, even for us in Microsoft.  I have made it much easier by simplifying the logs and importing them into Excel, color coding, etc. for both MS Support and our customers.
https://microsoft.github.io/CSS-Exchange/Calendar/Get-CalendarDiagnosticObjectsSummary/

**Comment by dfinke** [comment]
That is sweet, thank your for sharing this!

I don't work with Exhange.

Could you share a visual (or a sample xlsx) of the output? Would it be ok if I promoted this on the social networks?

Thank you again.

**Reply by Shanefe** [reply]
Sure, here is a sample XLS from 2 users.
[Demo1_CalLogSummary_7A9583.xlsx](https://github.com/user-attachments/files/17263379/Demo1_CalLogSummary_7A9583.xlsx)

![image](https://github.com/user-attachments/assets/c906b80f-b1ee-408b-8702-39cda5b8fdc1)

Share away.  
The above doc's will link to MS support articles on how to use this script on my next check in.
https://learn.microsoft.com/en-us/exchange/troubleshoot/calendars/get-calendar-diagnostic-logs  
https://learn.microsoft.com/en-us/exchange/troubleshoot/calendars/analyze-calendar-diagnostic-logs

**Reply by dfinke** [reply]
Terrific - Thank you!
